src/dyad/common/dyad_profiler.h

```diff
@@ -6,79 +6,84 @@
 #error "no config"
 #endif
 
-#ifdef DYAD_PROFILER_DLIO_PROFILER // DLIO_PROFILER
+#ifdef DYAD_PROFILER_DLIO_PROFILER  // DLIO_PROFILER
```

Comment:
We should do the formatting part of different PR.

Reply:
We already have `.clang-format` to handle formatting, although I've seen plenty of cases where it's not consistent. Not sure why at this point.